### DISCUSSION TITLE
[URGENT] Github Pages has suddenly stopped updating (no longer syncing with repo)

### DISCUSSION THREAD

**Discussion by josephrocca** [community/community#51349] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Here are two Github pages URLs that should work (i.e. shouldn't return a 404):

* https://josephrocca.github.io/OpenCharacters/test.txt
* https://josephrocca.github.io/onnxscript-editor/demo/test.txt

Here are their respective, associated repo files:

* https://github.com/josephrocca/OpenCharacters/blob/main/test.txt
* https://github.com/josephrocca/onnxscript-editor/blob/main/demo/test.txt

So the files exist, but they're not propagating to Github changes. Also, updates to existing files don't propagate either. It's stuck serving old versions of the files. The updates stopped propagating some time within the last couple of hours.

@queenofcorgis Apologies for the ping, and the [URGENT] tag, but this has been working fine for years, and this is a show-stopping bug, so I figured I'd try to get it escalated ASAP.

**Comment by LukeSavefrogs** [comment]
Do you use **Github Actions** to update the files? 

In that case this is related to #51350

**Reply by josephrocca** [reply]
Nope, I just set it up with the option in the repo settings, but it looks like you're right that it's related: https://www.githubstatus.com/

Thanks!

**Comment by tpak** [comment]
Status page says pages is degraded in addition to Actions: https://www.githubstatus.com/

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! Due to the current incident - Pages builds are paused. Please hang tight - we appreciate your patience!
